test/pushnotificationstestutils.h

```diff
@@ -30,9 +31,21 @@ class FakeWebSocketServer : public QObject
 
     ~FakeWebSocketServer();
 
+    QWebSocket *authenticateAccount(const OCC::AccountPtr account);
+
     void close();
 
-    static OCC::AccountPtr createAccount();
+    void waitForTextMessages() const;
+
+    uint32_t getTextMessagesCount() const;
+
+    QString getTextMessage(uint32_t messageNumber) const;
+
+    QWebSocket *getSocketForTextMessage(uint32_t messageNumber) const;
```

Comment:
socketForMessageNumber (this is not the text message you're passing)

Reply:
But it's somehow related to the text messages because there could be also binary messages. So I think just using `socketForMessage` would be ambiguous. What do you think about `socketForTextMessageByNumber`? But then `textMessage` should be renamed to `textMessageByNumber` I guess?